serverless-plugin/index.js

```diff
@@ -282,19 +265,20 @@ class ServerlessPlugin {
             const matches = handlerPattern.exec(func.handler);
 
             if (!matches) {
-                throw new Exception(`handler ${func.handler} was not of the form 'packageName.executableName' or 'dir1/dir2/packageName.executableName'.`);
+                throw new Exception(`handler ${func.handler} was not of the form 'ModuleName.functionName' or 'dir1/dir2/ModuleName.functionName'.`);
```

Comment:
Seems I misunderstood the syntax of `handler` - `src/` likely isn't needed in the examples, as `dir1/dir2/...` is the path to the _Stack project_, not the source file.

Reply:
With the runtime, it becomes the source file, not the project, like in JS